ci-operator/step-registry/baremetal/lab/ipi/install/baremetal-lab-ipi-install-commands.sh

```diff
@@ -96,21 +101,53 @@ platform:
     externalMACAddress: $(<"${SHARED_DIR}/ipi_bootstrap_mac_address")
     hosts: []
 "
+  yq --inplace eval-all 'select(fileIndex == 0).platform.baremetal = select(fileIndex == 1) | select(fileIndex == 0)' \
+    "$SHARED_DIR/install-config.yaml" - <<< "$ADAPTED_YAML"
+else
+  ADAPTED_YAML="
+    libvirtURI: >-
+      qemu+ssh://root@${AUX_HOST}:$(<"${CLUSTER_PROFILE_DIR}/provisioning-host-ssh-port-${architecture}")/system?keyfile=${CLUSTER_PROFILE_DIR}/ssh-key&no_verify=1&no_tty=1
+    apiVIP: $(yq ".api_vip" "${SHARED_DIR}/vips.yaml")
+    ingressVIP: $(yq ".ingress_vip" "${SHARED_DIR}/vips.yaml")
+    externalMACAddress: $(<"${SHARED_DIR}/ipi_bootstrap_mac_address")
+    provisioningNetwork: Disabled
+    hosts: []
+"
+  yq --inplace eval-all 'select(fileIndex == 0).platform.baremetal = select(fileIndex == 1) | select(fileIndex == 0)' \
+    "$SHARED_DIR/install-config.yaml" - <<< "$ADAPTED_YAML"
+fi
 
 echo "[INFO] Processing the platform.baremetal.hosts list in the install-config.yaml..."
 # shellcheck disable=SC2154
 for bmhost in $(yq e -o=j -I=0 '.[]' "${SHARED_DIR}/hosts.yaml"); do
   # shellcheck disable=SC1090
   . <(echo "$bmhost" | yq e 'to_entries | .[] | (.key + "=\"" + .value + "\"")')
+  if [ "${ENABLE_PROVISIONING_NETWORK:-}" == "true" ]; then
+    boot_mac="${provisioning_mac}"
+  else
+    boot_mac="${mac}"
+  fi
+
+  if [ "${REDFISH_VIRTUAL_MEDIA:-}" == "true" ]; then
```

Comment:
We should not overwrite variables obtained from the inventory. Also, we'd already have columns to catch these data. (look for the redfish_ columns or add new ones if needed).

Finally, as this change will impact different models, can you test it on the hpe arm64 servers too?